### ISSUE TITLE
"INSTALL" does not actually document the proper steps needed to install (from a Git clone or snapshot)

### ISSUE THREAD

**Issue by Ploy-XIV** [cpptest/cpptest#14]
The simplest sequence of steps that I found to install cpptest 2.0.0 when starting out from the cloned git repository at the tag `2.0.0` is the following:

~~~
./autogen.sh
./configure
make
make install
~~~

Meanwhile, `INSTALL` assumes that the script `configure` is part of the software distribution, which it is not.

My usual approach of running `autoreconf --install`, which typically figures out what parts of a GNU build system are missing (such as the `configure` script) does not work here, because the file `README` is missing (`autogen.sh` touches `README` in the steps above).

**Comment by hartwork** [comment]
Hi, thanks for getting in touch about this matter!

I have created a pull request with a proposal on how we could address this issue in detail at #15. Do you have a minute for review?

**Comment by Ploy-XIV** [comment]
Wow, that was quick, awesome!

I see you have removed the `INSTALL` altogether now in the pull request. I think it would still be good, if maybe `README` had a short note on what commands to run for a build. Looking at the repo again, I've noticed that the travis configuration file essentially also contains the four commands I posted in my issue description. So maybe the readme could contain a more generic: "To see the commands we use to build cpptest, consult `.travis.yml`", which will necessarily be up to date, if the CI makes a successful build.

Or, as an alternative, you could follow the [_GNU Coding Standards_ of the GNU Build System](https://www.gnu.org/software/automake/manual/automake.html#GNU-Build-System) and include a `configure` script with every distribution of the software, such as the downloadable zips/tars for each tagged version. Of course, this means more work for creating each release, just to save users the work of running `autogen.sh`...

~~Also, and this might be a bit of an unrelated issue, but `autoreconf` complains about a missing `README`, because only a `README.in` is present when the repo is first cloned. I was unsure when trying to build cpptest for the first time, whether or not this missing `README` now meant that `autoreconf` had not run properly. The auto-tools are still mostly black magic to me.~~
Never mind, the warning is gone with the pull request... my bad.

**Comment by hartwork** [comment]
> Or, as an alternative, you could follow the [_GNU Coding Standards_ of the GNU Build System](https://www.gnu.org/software/automake/manual/automake.html#GNU-Build-System) and include a `configure` script with every distribution of the software, such as the downloadable zips/tars for each tagged version. 

The 2.0.0 release downloads are a product of `make distcheck` and hence include a `configure` script, just a plain Git clone/snapshot does not. For an example, https://github.com/cpptest/cpptest/releases/download/2.0.0/cpptest-2.0.0.tar.bz2 contains a `configure` script, just re-checked.

> Also, and this might be a bit of an unrelated issue, but `autoreconf` complains about a missing `README`, because only a `README.in` is present when the repo is first cloned.

Thats fixed in #15 by moving from automake option `gnu` to `foreign`.

**Comment by hartwork** [comment]
> I see you have removed the `INSTALL` altogether now in the pull request. I think it would still be good, if maybe `README` had a short note on what commands to run for a build. Looking at the repo again, I've noticed that the travis configuration file essentially also contains the four commands I posted in my issue description. So maybe the readme could contain a more generic: "To see the commands we use to build cpptest, consult `.travis.yml`", which will necessarily be up to date, if the CI makes a successful build.

The current `README.in` has this:
```
Installation
============

See the files INSTALL.quick and INSTALL.
```
I'm open to pull requests to extend, inline, improve or add instructions to more places.

**Comment by Ploy-XIV** [comment]
> The 2.0.0 release downloads are a product of `make distcheck` and hence include a `configure` script, just a plain Git clone/snapshot does not. For an example, https://github.com/cpptest/cpptest/releases/download/2.0.0/cpptest-2.0.0.tar.bz2 contains a `configure` script, just re-checked.

Indeed. I only checked in the cloned repository, and not the tars...

> Thats fixed in #15 by moving from automake option `gnu` to `foreign`.

Thank you.
